Fix images on appstore page

lennartdohmann opened the pull request:
* Use plain markdown for images
* Resized images to a fixed height of 500px

unglaublicherdude commented:
Thank you for putting in the work. Wasn't here also some wording you wanted to change? Like to somwhere at least hint that you can also host vaas by yourself or did we already discard this?

lennartdohmann commented:
> Thank you for putting in the work. Wasn't here also some wording you wanted to change? Like to somwhere at least hint that you can also host vaas by yourself or did we already discard this?

I added it and also fixed a broken link to our registration page. @unglaublicherdude can you check if it is okay for you like that?
